src/tk/mem.h

```diff
@@ -146,7 +155,7 @@ void mem_checkptr(void *ptr);
  *              return NULL
  */
 
-void *mem_malloc(size_t);
+void *mem_malloc(d_size_t);
 void *mem_calloc(size_t);
```

Comment:
It's also not a problem leaving that as is for the moment, because it will show up as a linker error when it becomes a problem.